nvtabular/ops/hashed_cross.py

```diff
@@ -53,19 +54,19 @@ def __init__(self, num_buckets: Union[int, Dict[str, int]]):
         self.num_buckets = num_buckets
 
     @annotate("HashedCross_op", color="darkgreen", domain="nvt_python")
-    def transform(self, columns: ColumnNames, gdf: cudf.DataFrame) -> cudf.DataFrame:
-        new_gdf = cudf.DataFrame()
+    def transform(self, columns: ColumnNames, df: DataFrameType) -> DataFrameType:
+        new_df = type(df)({})
```

Comment:
```suggestion
        new_df = type(df)()
```
I don't think you need the empty dict here.